### ISSUE TITLE
Update to 1.0.4 fails with ' error: expected an item keyword'

### ISSUE THREAD

**Issue by adrianreber** [keeperofdakeys/dns-lookup#16]
The pull request from dependabot to upgrade dns-lookup to 1.0.4 fails with:
```
error: expected an item keyword
  --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/dns-lookup-1.0.4/src/lookup.rs:59:7
   |
59 |       unsafe {
   |       ^^^^^^
   | 
  ::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/cfg-if-0.1.9/src/lib.rs:41:40
   |
41 |         if #[cfg($($meta:meta),*)] { $($it:item)* }
   |                                        -------- while parsing argument for this `item` macro fragment

error: aborting due to previous error

error: could not compile `dns-lookup`.
```

Not sure if this is because of other dependencies or why it happens. The corresponding pull request is here: https://github.com/adrianreber/mirrorlist-server/pull/53

Is there anything I need to change in my project?

**Comment by keeperofdakeys** [comment]
Using `item` in that macro in cfg-if is wrong, it should be `tt`. The version failing here is cfg-if 0.1.9, and this issue appears to have been fixed in cfg-if 0.1.10 - https://github.com/alexcrichton/cfg-if/commit/88456aa305209154053057d17c3a4d41a2914709.

As cfg-if 1.0.0 has been released, I'll update the cargo.toml to use it and push another minor update.

**Comment by keeperofdakeys** [comment]
Okay I've pushed version 1.0.5 for this, let me know if you still have a build failure.

**Comment by adrianreber** [comment]
Thanks a lot for the quick fix. Errors are now resolved and I updated to dns-lookup 1.0.5.
